### DISCUSSION TITLE
Radio Paradise Problems

### DISCUSSION THREAD

**Discussion by R101G** [rern/rAudio#475] [Fixed Issues]
@rern 
The Radio Paradise library entries have changed, and do not seem to work.

**Comment by Roland-B** [comment]
> 
> 
> @rern
> The Radio Paradise library entries have changed, and do not seem to work.

@R101-Github 

Have just checked as of now (16/08/2021 15:04) and all the Radio Paradise stations work from Webradio Library for me, nothing appears to have changed as far as I can tell.

![image](https://user-images.githubusercontent.com/43496391/129576298-c788c508-b678-461b-bfee-5cc9b3e784c7.png)
![image](https://user-images.githubusercontent.com/43496391/129576866-489cda7f-a858-4dcc-9147-58470cffd07e.png)


RolandB

rAudio 1 • 20210813
5.10.52-7-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Reply by R101G** [reply]
@Roland-B 

Thanks for the reply. My backup copy of webradios has http and flacm addresses for RP. When I copied those in, RP worked again, but the coverart still has some intermittent problems.

I did a kernel update after applying the 20210813 update, so perhaps that messed something up. I'll wait to see if the next update sorts it out before trying a new image.

**Reply by Roland-B** [reply]
@R101-Github 

No problem. Just tested **"http://stream.radioparadise.com/flacm"** on mine and can confirm using that address the cover art also has issues for me also. Covers work fine using the the https flac links I currently have shown in the pic above. I'm not quite sure why flacm is needed tbh, as I get metadata with the standard flac address as can be seen in previous post or am I missing something?

RolandB

rAudio 1 • 20210813
5.10.52-7-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Comment by R101G** [comment]
@Roland-B 

The mystery deepens!   I went back to a 20210724 backup image, which has flacm URLs. This works, and if I change to the flac URLs I get sound, but no art, just as the RP website says.

I then updated with -Syu to 5.10.52-8-ARCH aarch64 and it still works, but when the 20210813 update is applied, the webradio files are overwritten, and the RP channels do not work at all. No sound, no data.

There must be something different about our setups which affects this. The problem with having such a configurable system is that no two systems are going to be exactly the same, which is great for the users but a nightmare for rern.

**Reply by Roland-B** [reply]
@R101-Github That is very odd!? I guess there must be some kind of difference in our setups. May possibly be your restore from backup? I have just installed fresh latest 64bit build ([3 days ago](https://github.com/rern/rAudio-1/releases/download/i20210813/rAudio-1-RPi64-20210813.img.xz)) to check, made no changes other than setting DAC as audio and enabling FFmpeg, and on looking the default addresses for Paradise Radio are the https and flac, and metadata is there and working. My guess is @rern changed these a while ago for some reason, and mine is working as I started from scratch not too long ago and didnt use a restore from backup? Other than that I have no idea and yup must be a nightmare for rern with so many possible combinations of setup/issues.. I salute that man!

Edit: Also just updated to 5.10.52-8-ARCH, and still remains working fine.

Edit2: Just looked at a backup file from last month (20210721) I have, and indeed the addresses have been changed in the build recently, my backup contains http flacm addresses.  

RolandB

rAudio 1 • 20210813
5.10.52-8-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Comment by R101G** [comment]
@Roland-B 
I make a backup image of the SD card whenever things are working well, but the original problem was a result of the 20210813 update. Reverting to the backup image and updating everything again was just to confirm. You can see the 20210813 update overwriting the webradio files at the start of the update. (Edit: I am using a backup image, not a restore from backup.)

I can't think why any settings would affect RP and none of the other streams, although I realise the coverart comes from a different source.

I bet your NTP time and date works too :)

**Reply by Roland-B** [reply]
Nope neither can I, hopefully rern can get to the bottom of it sooner rather than later, not a real prob as such not having a cover but I find these little things aggrivate my OCD :)  Yup indeed.. my date/time appears correct thankfully :)  

RolandB

rAudio 1 • 20210813
5.10.52-8-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Comment by R101G** [comment]
@Roland-B 

So it might be that your new image has the modifications which work, and my long-term updating doesn't. I will try a new build from scratch.

**Reply by Roland-B** [reply]
@R101-Github 

Okidoke, fingers crossed. I still think that there is an issue that possibly needs to be addressed as surely the flacm address should still also get a cover for both us no matter what build update we have if manually added (even though no reason it should be added though if current existing one works), even if the updated flac address now used works for getting one.

RolandB

rAudio 1 • 20210813
5.10.52-8-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Reply by rern** [reply]
URLs of Radio Paradise have been changed, `*flacm` to `*flac` recently.

`*flacm`s - There are 5 seconds interval of embedded metadata which cause unnecessary metadata fetching every 5 seconds.

**Comment by R101G** [comment]
@Roland-B @rern 

I tried a fresh install with the latest image. Neither of the bugs I have open occur. RP and the NTP server work perfectly. I assumed there would be no difference between applying updates to an older image and using the latest image, but that is clearly not so.

I am having serious problems getting wifi to connect automatically on the new image, so not quite out of the woods yet.

**Reply by Roland-B** [reply]
@R101-Github Glad you got the 2 issues worked out, hopefully you'll have the wifi sorted soon. Personally i've always thought wifi on this build was a bit flakey, but as I dont use I cant comment on how it is currently.  Fingers crossed for you R101-Github.

RolandBrAudio 1 • 20210813
5.10.52-8-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Comment by R101G** [comment]
@Roland-B 
Thanks for the support. It looks as if the driver for my wifi dongle needs updating, so I am using on-board Pi3 wifi for the moment, which is not good.
 
I will move this to solved.

Interestingly, I see the system time is now wrong.

**Comment by Roland-B** [comment]
@R101-Github Your very welcome. That is a weird one with the time!?  What difference/skew are you seeing in yours out of curiosity?

Just checked mine for you :
rAudio:~  #  date
Tue Aug 17 14:31:34 BST 2021

RolandB

rAudio 1 • 20210813
5.10.52-8-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Comment by R101G** [comment]
@Roland-B 
As I am using a temporary play image, and could not get anywhere with the installed timesyncd, I installed ntp from pacman. That works a treat, but I have probably messed up rern's good work, so will revert to normal when I know what it is I am doing to cause the problem. The time on the fresh install was solid for a while, but after a lot of reboots, I noticed it was wrong by 20 minutes or so.

**Reply by R101G** [reply]
Please see discussions #472 and #479

**Reply by Roland-B** [reply]
@R101-Github Cheers, explains why I never had an issue then as I only run mine via ethernet.

RolandB

rAudio 1 • 20210820
5.10.59-1-ARCH aarch64
Raspberry Pi 4 Model B Rev 1.2
BCM2711 • 3.7GiB
4 @ 1.5GHz

**Reply by R101G** [reply]
@Roland-B Yes, I think this time-sync problem might have caused some other problems too. I'm grateful to @teslasparks301 for finding the connection.
